index.d.ts

```diff
@@ -1,4 +1,28 @@
 declare module "svgpath" {
+  type MoveToAbs = ["M", number, number];
+  type LineToAbs = ["L", number, number];
```

Comment:
All comands can be repeated (have more parems). For 2 lines: `["L", number, number, number, number]`